tests/data.py

```diff
@@ -7,6 +7,7 @@ class Actor(Enum):
     RCA = ("555031998104", "D82106", "REFERRING_CLINICIAN_ADMIN")
     SPC = ("555032006106", "RCD", "SERVICE_PROVIDER_CLINICIAN")
     SPCA = ("555031998104", "RCD", "SERVICE_PROVIDER_CLINICIAN_ADMIN", "555031999105")
+    AA = ("555031998104", "RCD", "AUTHORISED_APPLICATION")
```

Comment:
I don't think this user would ever be valid currently, I wonder if its worth including a comment highlighting that this actor isn't usable for user-restricted access.